src/osm_osw_reformatter/serializer/osm/osm_normalizer.py

```diff
@@ -45,18 +45,16 @@ def filter_tags(self, tags):
             tags.pop('foot', '')
 
         # OSW fields with similar OSM field names
+        if 'highway' in tags and tags['highway'] != 'steps':
+            tags.pop('climb', '')
```

Comment:
check if `climb` tag is `up` or `down`. If none of these two, probably its best to ignore the value